src/elements/content-sidebar/activity-feed/annotation-thread/__tests__/useReplies.test.js

```diff
@@ -0,0 +1,145 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+import { annotation, user } from '../../../../../__mocks__/annotations';
+import useReplies from '../useReplies';
+import { replies } from '../__mocks__/replies';
+
+describe('src/elements/content-sidebar/activity-feed/useReplies', () => {
+    let mockCreateAnnotationReply = jest.fn();
```

Comment:
I always recommend avoiding sharing variables between tests, as it can leak state if they're not reset properly.